walkthroughs/1-integrating-event-and-api-driven-apps/walkthrough.adoc

```diff
@@ -1,39 +1,40 @@
 // tag::master-1[]
-:walkthrough: Integrating event-driven and API-driven applications (AMQ)
+:walkthrough: Integrating message-oriented middleware with an API using AMQ
+:messaging-service: Red Hat AMQ
 :fuse-version: 7.2
+:AMQ-ProductLongName: Red Hat AMQ
+:AMQ-BrokerVersion: 7.2
+:Fuse-prodnamefull: Red Hat Fuse
```

Comment:
Not 100% sure but is the product name Red Hat Fuse or Red Hat Fuse Online?

Reply:
changing to Red Hat Fuse Online based on https://www.redhat.com/en/technologies/jboss-middleware/fuse-online